embeddings/embedding/static/word2vec.py

```diff
@@ -58,3 +59,49 @@ def from_default_config(
     @staticmethod
     def get_config(**kwargs: Any) -> KGR10Word2VecConfig:
         return KGR10Word2VecConfig(**kwargs)
+
+
+@dataclass
+class IPIPANWord2VecConfig(GensimLocalFileConfig):
+    corpus: str = field(init=False)
+    type: str = field(init=False)
+    subtype: str = field(init=False)
+    dimension: int = field(init=False)
+    method:  str = field(init=False)
+    algorithm:  str = field(init=False)
+    model_name: str = field(init=False)
+    reduced_vocabulary: bool = False
+    model_type_reference: str = "embeddings.embedding.static.word2vec.IPIPANWord2VecEmbedding"
+
+    def __post_init__(self) -> None:
+        self.model_name = self.model_file_path.name.split(".")[0]
+
+        metadata = self.model_name.split("-")
+        assert len(metadata) in [
```

Comment:
```suggestion
        assert 6 <= len(metadata) <= 8
```
I think also a comment here why it is not consistent would be helpful ;)